app-emulation/vagrant/vagrant-2.2.14.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+USE_RUBY="ruby25 ruby26"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="vagrant.gemspec"
+RUBY_FAKEGEM_EXTRAINSTALL="keys plugins templates version.txt"
+RUBY_FAKEGEM_TASK_DOC=""
+
+inherit bash-completion-r1 ruby-fakegem
+
+DESCRIPTION="A tool for building and distributing development environments"
+HOMEPAGE="https://vagrantup.com/"
+SRC_URI="https://github.com/hashicorp/vagrant/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+virtualbox test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="${RDEPEND}
+	app-arch/libarchive
+	net-misc/curl
+	virtualbox? ( || ( app-emulation/virtualbox ) )"
+
+ruby_add_rdepend "
+	>=dev-ruby/bcrypt_pbkdf-1.0.0
```

Comment:
@GuillaumeSeren  Have you tested this with `dev-ruby/bcrypt_pbkdf-1.1.0`?  I am seeing problems with the current version of vagrant not working with this version.  Not sure if I can be patched up in the gemspec or if there is really some incompatibility.

Reply:
Hey @kwhat ,
I just tested both 1.0.1 and 1.1.0 both seems to works, it only require a small gemspec upgrade.
As I require '>=dev-ruby/bcrypt_pbkdf-1.0.0', I will upgrade the gemspec as well.

Reply:
I also tested yesterday and came to the same conclusion, thanks!